AmplifyPlugins/Analytics/AWSPinpointAnalyticsPluginTests/Pinpoint/EndpointClientTests.swift

```diff
@@ -17,20 +17,13 @@ class EndpointClientTests: XCTestCase {
     private var device: MockDevice!
     private var currentApplicationId = "applicationId"
     private var currentEndpointId = "endpointId"
-
+    private var keychain: MockKeychainStore!
     override func setUp() {
         archiver = MockArchiver()
         userDefaults = MockUserDefaults()
         pinpointClient = MockPinpointClient()
         device = MockDevice()
-        endpointClient = EndpointClient(configuration: .init(appId: currentApplicationId,
```

Comment:
Why was this removed from the `setUp` and added to all tests instead?

Reply:
Oh got it! 

If you face a similar situation in the future, there is an `async` version of the `setUp` function as well for tests, so you can just override that one if you need to await for other methods :)